tensorflow/core/api_def/base_api/api_def_QuantizedMatMulWithBias.pbtxt

```diff
@@ -1,4 +1,85 @@
 op {
   graph_op_name: "QuantizedMatMulWithBias"
   visibility: HIDDEN
+  in_arg {
+    name: "a"
+    description: <<END
+Must be a two-dimensional tensor.
```

Comment:
Please explain what `a` is (matrix to be multiplied, etc). The same goes for `b`.
Also, for anything that fits in one line, please use double quotes instead, i.e.,
`description: "Must be a two-dimensional tensor."`